### DISCUSSION TITLE
Encryption: How to open files on a computer?

### DISCUSSION THREAD

**Discussion by gsantner** [gsantner/markor#1372] [Q&A] [answered]
Discussion moved from other thread:  
![grafik](https://user-images.githubusercontent.com/6735650/122775318-4ba04500-d2aa-11eb-9c85-0bc222c894a9.png)

**Comment by gsantner** [comment]
* <https://github.com/gsantner/markor>
* <https://gitlab.com/opensource21/jpencconverter>

**Reply by cser** [reply]
Is it possible to use OpenSSL directly?

**Reply by gsantner** [reply]
if you can figure out how to generate/strip the proper metadata/leading header, probably. Please ask at jpencconverter project.

**Comment by clach04** [comment, marked as answer]
I just implemented support for jenc format in https://github.com/clach04/puren_tonbo/issues/169 so can encrypt/decrypt single files on the command line.

    python -m puren_tonbo.tools.ptcipher Test3.md.jenc -p geheim

Along with an encrypted grep tool. NOTE this is NOT a GUI, so targetted at people comfortable with both command line and Python.
